contribs/drt/src/main/java/org/matsim/contrib/drt/passenger/MaxDetourOfferAcceptor.java

```diff
@@ -0,0 +1,23 @@
+package org.matsim.contrib.drt.passenger;
+
+import java.util.Optional;
+
+public class MaxDetourOfferAcceptor implements DrtOfferAcceptor{
+	private final double promisedPickupTimeWindow;
+
+	public MaxDetourOfferAcceptor(double promisedPickupTimeWindow) {
+		this.promisedPickupTimeWindow = promisedPickupTimeWindow;
+	}
+
+	@Override
+	public Optional<AcceptedDrtRequest> acceptDrtOffer(DrtRequest request, double departureTime, double arrivalTime) {
+		double updatedPickupTimeWindow = Math.min(departureTime
+			+ promisedPickupTimeWindow, request.getLatestStartTime());
```

Comment:
Consider a different name than `promisedPickupTimeWindow`. We usually think of a time window as a `[a,b)` period, where `a` and `b` are time stamps. Here, `promisedPickupTimeWindow` is a duration and represents a time buffer.

In the drt config group, we use `maxAllowedPickupDelay` instetad of `promisedPickupTimeWindow`. I think we should also use this name in this class.

Reply:
Also `updatedPickupTimeWindow` should be renamed. Maybe `updatedLatestStartTime`?